src/Analysis/GWASResults/Views/Input/JobDetails.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { useContext } from 'react';
+import { useQuery } from 'react-query';
+import { Spin } from 'antd';
+import { gwasWorkflowPath } from '../../../../localconf';
+import IsJsonString from '../../Utils/IsJsonString';
+import SharedContext from '../../Utils/SharedContext';
+import LoadingErrorMessage from '../../SharedComponents/LoadingErrorMessage/LoadingErrorMessage';
+
+const JobDetails = () => {
+  const { selectedRowData } = useContext(SharedContext);
+  const { name, uid } = selectedRowData;
+  const endpoint = `${gwasWorkflowPath}status/${name}?uid=${uid}`;
+
+  const fetchData = async () => {
+    const res = await fetch(endpoint);
+    return res.json();
+  };
+
+  const { data, status } = useQuery('jobDetails', fetchData);
+
+  if (status === 'error') {
+    return <LoadingErrorMessage message='Error getting job details' />;
+  }
+  if (status === 'loading') {
+    return (
+      <div className='spinner-container' data-testid='spinner'>
+        <Spin />
+      </div>
+    );
+  }
+
+  if (!data || data.length === 0 || data.error) {
+    return <LoadingErrorMessage message='Issue Loading Data for Job Details' />;
+  }
+
+  const getParameterData = (key) => {
+    const datum = data?.arguments?.parameters?.find((obj) => obj.name === key);
+    return datum?.value || 'Data not found';
+  };
+
+  const getPhenotype = () => {
+    if (
+      getParameterData('outcome')
+      && IsJsonString(getParameterData('outcome'))
+    ) {
+      return (
+        JSON.parse(getParameterData('outcome'))?.concept_name
+        || JSON.parse(getParameterData('outcome'))?.provided_name
+      );
+    }
+    return 'Data not found';
+  };
+
+  const processCovariates = () => {
+    const input = JSON.stringify(getParameterData('variables'));
+    let covariatesString = input.replaceAll('\\n', '');
+    covariatesString = covariatesString.substring(
+      1,
+      covariatesString.length - 1,
+    );
+    const strToRemove = '\\"';
```

Comment:
why is this needed? Is our argo workflow code or argo-wrapper doing something strange with this data? I'm wondering if we can try to fix that in argo-wrapper itself.